docs/design/ovs-pipeline.md

```diff
@@ -707,116 +775,143 @@ priority than K8s default drop flows. For example, a baseline rule to drop
+    since it is no need to decrease TTL value of packets from connections among local Pods.
+- Flow 4 matches packets destined to remote Pod CIDR. This means that we install one flow for each peer Node, each one
+  matching the destination IP address of the packet against the Pod subnet for the Node. 
+  - Match condition `nw_dst=10.10.1.0/24` is to match packets destined to remote Pod CIDR.
+  - Action `set_field:ba:5e:d1:55:aa:c0->eth_src` is to rewrite the source MAC address of packets to the local gateway
+    MAC address.
+  - Action `set_field:aa:bb:cc:dd:ee:ff->eth_dst` is to rewrite the destination MAC address of packets to the Global
+    Virtual MAC address.
+  - Action `set_field:192.168.77.103->tun_dst` is to set destination IP address (i.e. the IP address of the remote gateway)
+    of tunnel to remote Node.
+  - Action `set_field:0x10/0xf0->reg0` is to load `ToRemoteRegMark` (bits 4-7 of `NXM_NX_REG0` is 0x1), which indicates
+    that the output port is tunnel.
+- Flow 5-8 matches packets destined to local Pods. The packets could be from Service or inter-Node connections.
+  - Match condition `reg0=0x200/0x200` is to match `RewriteMACRegMark` (bit 9 of `NXM_NX_REG0` is 0x1), indicating that
+    the destination and source MACs of packets should be overwritten. This is to match Service or inter-Node packets
+    from connections.
+  - Match condition `nw_dst=<POD_IP_ADDRESS>` is to match packets whose destination IP is the IP address of a local Pod.
+  - Action `set_field:ba:5e:d1:55:aa:c0->eth_src` is to rewrite the source MAC address of packets to the local gateway
+    MAC address.
+  - Action `set_field:<POD_MAC_ADDRESS>->eth_dst` is to rewrite the destination MAC address of packets to the local Pod
+    MAC address.
+  - Action `goto_table:L3DecTTL` is the same as flow 1.
+- Flow 9 is to match packets of Service connections sourced from local Antrea gateway and destined to external network.
+  - Match condition `ct_mark=0x10/0x10` is to match `ServiceCTMark` (bit 4 of ct mark is 0x1), indicating packets are
+    from Service connections.
+  - Match condition `reg0=0x202/0x20f` is to match `RewriteMACRegMark` (bit 9 of `NXM_NX_REG0` is 0x1) and
+    `FromGatewayRegMark` (bits 0-3 of `NXM_NX_REG0` is 0x2), indicating packets are from Service connections originated
+    from local Antrea gateway.
+  - Action `set_field:ba:5e:d1:55:aa:c0->eth_dst` is to rewrite the destination MAC address of packets to the local
+    gateway MAC address.
+  - Action `set_field:0x20/0xf0->reg0` is to load `ToGatewayRegMark` (bits 4-7 of `NXM_NX_REG0` is 0x2), which indicates
+    that the output port is local Antrea gateway.
+  - Action `goto_table:L3DecTTL` is the same as flow 1.
+- Flow 10 is to match packets that are from local Pods and destined to external network, and send them to table [EgressMark],
+  where SNAT IPs are looked up for the packets.
```

Comment:
```suggestion
  where SNAT are performed on the packets.
```

Reply:
Egress SNAT is not performed in OVS pipeline, and it is performed by iptables.

Reply:
what about 
```
and send them to table [EgressMark] to find the SNAT IPs.
```